public/index.html

```diff
@@ -24,12 +24,16 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
-  </head>
-  <body>
-    <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root"></div>
-    <!--
+		<script
+			src="https://kit.fontawesome.com/9b9e4cb61d.js"
+			crossorigin="anonymous"
+		></script>
+		<title>React App</title>
```

Comment:
Provide good title name for your website

Reply:
Yes, Changed it